syntax = "proto3";

package selserv;

option go_package = "github.com/FishyFish3r/crypto-parser/pkg/selserv";

service SeleniumServer {
    rpc GetHtml(HtmlArgs) returns (HtmlResponse);
}

message HtmlArgs {
    string Url = 1;
}

message HtmlResponse {
    string Html = 1;
}